tests/puppeteer/pages/BO/orders/invoices/index.js

```diff
@@ -20,22 +20,25 @@ module.exports = class Invoice extends BOBasePage {
     // By order status form
     this.generateByStatusForm = '[name="generate_by_status"]';
     this.formGenerateByStatus = '#form_generate_by_status_order_states';
-    this.statusOrderStateInput = `${this.formGenerateByStatus} input#form_generate_by_status_order_states_%ID`;
-    this.statusCheckbox = `${this.statusOrderStateInput}:first-of-type + i`;
+    this.statusOrderStateInput = id => `${this.formGenerateByStatus} input#form_generate_by_status_order_states_${id}`;
+    this.statusCheckbox = id => `${this.statusOrderStateInput(id)}:first-of-type + i`;
     this.generatePdfByStatusButton = `${this.generateByStatusForm} .btn.btn-primary`;
     // Invoice options form
     this.invoiceOptionsForm = '[name="invoice_options"]';
-    this.invoiceOptionsEnable = `${this.invoiceOptionsForm} label[for="form_invoice_options_enable_invoices_%ID"]`;
-    this.taxBreakdownEnable = `${this.invoiceOptionsForm} label[for="form_invoice_options_enable_tax_breakdown_%ID"]`;
-    this.invoiceOptionEnableProductImage = `${this.invoiceOptionsForm} 
-    label[for="form_invoice_options_enable_product_images_%ID"]`;
+    this.invoiceOptionsEnable = id => `${this.invoiceOptionsForm
+    } label[for='form_invoice_options_enable_invoices_${id}']`;
```

Comment:
This is the correct way to break the line ? :fearful:

Reply:
I made this modification 
```js
        this.invoiceOptionsEnable = id => this.invoiceOptionsForm 
          + ` label[for='form_invoice_options_enable_invoices_${id}']`;
```

But eslint change it.